src/test/CMakeLists.txt

```diff
@@ -659,39 +666,58 @@ set_target_properties(
     INSTALL_RPATH ""
 )
 
-target_include_directories(
-  input_byte_size_test
-  PRIVATE
-    ${CMAKE_CURRENT_SOURCE_DIR}/..
-    ${CMAKE_CURRENT_SOURCE_DIR}/../../include
-    ${GTEST_INCLUDE_DIRS}
-    ${CNMEM_PATH}/include
-)
+if(${TRITON_ENABLE_GPU})
```

Comment:
My preference would be to merge this if/else with the if/else above. Doing so would also make it more consistent with the rest of the file.

Reply:
My test cases are partially run in CPU.

Reply:
Right, but I think it can be condensed into a single if/else block for clarify. I'll leave it up to you.

Reply:
I agree. Fixed.